src/NzbDrone.Core/Download/Clients/NzbVortex/NzbVortex.cs

```diff
@@ -0,0 +1,263 @@
+        {
+            try
+            {
+                _proxy.GetVersion(Settings);
+            }
+            catch (Exception ex)
+            {
+                _logger.ErrorException(ex.Message, ex);
+                return new ValidationFailure("Host", "Unable to connect to NZBVortex");
+            }
+
+            return null;
+        }
+
+        private ValidationFailure TestApiVersion()
+        {
+            try
+            {
+                var response = _proxy.GetApiVersion(Settings);
+                var version = new Version(response.ApiLevel);
+
+                if (version.Major < 2 || (version.Major == 2 && version.Minor < 3))
+                {
+                    return new ValidationFailure("Host", "NZBVortex needs to be updated");
+                }
+            }
+            catch (Exception ex)
+            {
+                _logger.ErrorException(ex.Message, ex);
+                return new ValidationFailure("Host", "Unable to connect to NZBVortex");
+            }
+
+            return null;
+        }
+
+        private ValidationFailure TestAuthentication()
+        {
+            try
+            {
+                _proxy.GetQueue(1, Settings);
+            }
+            catch (NzbVortexAuthenticationException ex)
+            {
+                return new ValidationFailure("ApiKey", "API Key Incorrect");
+            }
+
+            return null;
+        }
+
+        private ValidationFailure TestCategory()
+        {
+            var group = GetGroups().FirstOrDefault(c => c.GroupName == Settings.TvCategory);
+
+            if (group == null)
+            {
+                if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+                {
+                    return new NzbDroneValidationFailure("TvCategory", "Group does not exist")
+                    {
+                        DetailedDescription = "The Group you entered doesn't exist in NzbVortex. Go to NzbVortex to create it."
+                    };
+                }
+            }
+
+            return null;
+        }
+
+        private OsPath GetOutputPath(NzbVortexQueueItem vortexQueueItem, DownloadClientItem queueItem)
+        {
+            var outputPath = _remotePathMappingService.RemapRemoteToLocal(Settings.Host, new OsPath(vortexQueueItem.DestinationPath));
+
+            if (outputPath.FileName == vortexQueueItem.UiTitle)
+            {
+                return outputPath;
+            }
+
+            // If the release isn't done yet, skip the files check and return null
+            if (vortexQueueItem.State != NzbVortexStateType.Done)
+            {
+                return new OsPath(null);
+            }
+
+            var filesResponse = _proxy.GetFiles(vortexQueueItem.Id, Settings);
+
+            if (filesResponse.Files.Count > 1)
+            {
+                queueItem.Status = DownloadItemStatus.Warning;
+                _logger.Warn("Download contains multiple files and is not in a job folder: {0}", outputPath);
```

Comment:
This will get logged every minute.
Override the ```Message``` so it shows up on the Queue UI, and log it at Debug/Trace.